README.md

```diff
@@ -155,31 +155,38 @@ const example2: MomoJob = {
 const example3: MomoJob = {
   name: 'example 3',
   interval: '5 minutes',
-  delay: 60 * 1000, // 1 minute
+  firstRunAfter: 60 * 1000, // 1 minute
   handler: () => console.log('This is momo'),
 };
 
 const example4: MomoJob = {
   name: 'example 4',
   interval: '5 minutes',
-  delay: 10 * 60 * 1000, // 10 minutes
+  firstRunAfter: 5 * 60 * 1000, // 5 minutes
+  handler: () => console.log('This is momo'),
+};
+
+const example5: MomoJob = {
+  name: 'example 5',
+  interval: '5 minutes',
+  firstRunAfter: 10 * 60 * 1000, // 10 minutes
   handler: () => console.log('This is momo'),
 };
 ```
 
-Assume it is 12:00 AM when the MongoSchedule with these four example jobs is started.
+Assume it is 12:00 AM when the MongoSchedule with these five example jobs is started.
 
-- `example 1` will be run immediately, at 12:00, and then every five minutes.
-- `example 2` will be run after 5 minutes (the configured interval), at 12:05, and then every five minutes.
-- `example 3` will be run after 1 minute (the configured delay), at 12:01, and then every five minutes.
-- `example 4` will be run after 10 minutes (the configured delay), at 12:10, and then every five minutes.
+- `example 1` and `example 2` are equivalent, since `firstRunAfter` defaults to `0`. They will be run immediately, at 12:00, and then every five minutes.
+- `example 3` will be run after 1 minute (the configured `firstRunAfter`), at 12:01, and then every five minutes.
+- `example 4` will be run after 5 minutes (the configured `firstRunAfter`), at 12:05, and then every five minutes.
+- `example 5` will be run after 10 minutes (the configured `firstRunAfter`), at 12:10, and then every five minutes.
 
 Now assume the MongoSchedule is stopped at 12:02 and then immediately started again.
 
-- `example 1` will be run 5 minutes (the configured interval) after the last execution, at 12:05. The job is not run immediately because it already ran before.
-- `example 2` will be run 5 minutes (the configured interval) after the start, at 12:07.
-- `example 3` will be run 5 minutes (the configured interval) after the last execution, at 12:06. The job is not run immediately because it already ran before.
-- `example 4` will be run after 10 minutes (the configured delay), at 12:12, and then every five minutes.
+- `example 1` and `example 2` will be run 5 minutes (the configured `interval`) after the last execution, at 12:05. They are NOT run immediately because they already ran before.
+- `example 3` will be run 5 minutes (the configured `interval`) after the last execution, at 12:06. The job is NOT run after 1 minute (the configured `firstRunAfter`) because it already ran before.
+- `example 4` will be run 5 minutes (the configured `firstRunAfter`) after the start, at 12:07, because it never ran before, and then every five minutes.
```

Comment:
I think we can remove example 4, it is the same as 5.

Reply:
My thought was just to demonstrate statt firstRunAfter can be different than interval (of course that's quite obvious, but for people looking mainly at examples ...(which leads me to the question, should we maybe move up the examples?!)) -
but you are right, in this second part with the restartet schedule it actually makes no difference. I'll see if I can "merge" ex. 4+5 here